debian/extra_files/40-hardware-id.toml

```diff
@@ -1,2 +1,2 @@
 [provision]
-primary_ecu_hardware_id = "ubuntu-20-04"
+primary_ecu_hardware_id = "ubuntu-focal"
```

Comment:
Please drop this commit (see TCCP-733 on Jira).

Reply:
ack